package.json

```diff
@@ -96,7 +98,10 @@
 			"plugin:eslint-plugin/all"
 		],
 		"ignores": [
-			"test/integration/{fixtures,unicorn}/**"
+			"test/integration/{fixtures,unicorn}/**",
+			"test/smoke/**",
```

Comment:
Why are we ignoring this file?

Reply:
Ignoring removed. Applied required styling to contents of `test/smoke/**`.